doc/15-troubleshooting.md

```diff
@@ -1,5 +1,51 @@
 # Icinga 2 Troubleshooting <a id="troubleshooting"></a>
 
+## Icinga starts/restarts/reloads very slowly
+
+### If you run Linux and systemd
```

Comment:
I wouldn't have this and "If it's faster with libjemalloc" as heading but rather incorporate it into the text. On first glance, this might look like these could be alternatives and also having them show up in the table of contents isn't really helpful either.